stories/KeyListener.stories.tsx

```diff
@@ -11,8 +13,18 @@ export default {
 
 type Story = StoryObj<StoryProps>;
 
-export const Default: Story = {
-  args: {
-    children: 'KeyListener',
+export const onKeyDown: Story = {
+  render: () => {
+    useEffect(() => {
+      setupHotKeys();
+    }, []);
+
+    return (
+      <KeyListener
+        onKeyDown={(e: KeyEvent) => {
+          console.log('onKeyDown', e.toString(), e);
```

Comment:
Make sure to add the withConsole decorator like input has

Reply:
Not necessary

Reply:
Where are the console logs showing?